t/dbi.t

```diff
@@ -418,6 +418,36 @@ for my $spec (
         dbi => [ [Server => 'foo'], [Port => 123], [Database => 'try'] ],
         qry => [ foo => 1, foo => 2, lol => 'yes', Driver => 'Snowflaker' ],
     },
+    {
+        uri => 'db:clickhouse:',
+        dsn => 'dbi:ODBC:',
+        dbi => [ [DSN => undef] ],
+        qry => [],
+    },
+    {
+        uri => 'db:clickhouse:dbadmin',
+        dsn => 'dbi:ODBC:DSN=dbadmin',
+        dbi => [ [DSN => 'dbadmin'] ],
+        qry => [],
+    },
+    {
+        uri => 'db:clickhouse://yow',
+        dsn => 'dbi:ODBC:Server=yow;Port=8123',
+        dbi => [ [Server => 'yow'], [Port => 8123], [Database => undef] ],
```

Comment:
This seems fine to me if these typical ODBC config params are what click house's ODBC driver in fact supports. I don't see any param docs in the [clickhouse-odbc](/yandex/clickhouse-odbc) project page, but I'm happy to take your word for it.